JustinDavis.txt

```diff
@@ -0,0 +1,9 @@
+
+Hello, My name is Justin Davis.
+
+Bears
+Beets
+Battlestar Galactica
+
+   My favorite hobbies include; making music, working on games, coding, and design. I like these things because they all
+work hand-in-hand but don't force me to specialize. "Specialization is for insects." � Robert Heinlein 
```

Comment:
Battlestar Galactica is great. And beets are a super-food that people should be eating more of. I read about a village in the Ukraine where people often live to be 100, and their diet is very beet-heavy.